apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -63,3 +80,35 @@ export const WarningLabel = ({children}) => (
 WarningLabel.propTypes = {
   children: React.PropTypes.node
 };
+
+/**
+ * Given a search query, generate a results list of animationProps objects that
+ * can be displayed and used to add an animation to the project.
+ * @param {string} searchQuery - text entered by the user to find an animation
+ * @return {Array.<SerializedAnimationProps>} - roughly
+ */
+function searchAnimations(searchQuery) {
+  // Make sure to generate the search regex in advance, only once.
+  // Search is case-insensitive
+  // Match any word boundary or underscore followed by the search query.
+  // Example: searchQuery "bar"
+  //   Will match: "barbell", "foo-bar", "foo_bar" or "foo bar"
+  //   Will not match: "foobar", "ubar"
```

Comment:
The 1000-10000 was my own ballpark, I'm not sure what the content team has in mind.  I think kids will know what to search for but there is a discoverability concern - I was actually wondering how we should special-case the results when there is no search query.  Right now we always display the first n animations (alphabetical) but I'm wondering if we should instead pick a random set, or highlight the newest content.